lib/chai.js

```diff
@@ -4,14 +4,13 @@
  * MIT Licensed
  */
 
-var used = []
-  , exports = module.exports = {};
+var used = [];
 
 /*!
  * Chai version
  */
 
-exports.version = '3.5.0';
+exports.version = require('../package').version;
```

Comment:
Yeah, when bundling for browsers result package size is an issue. We can use Rollup to decrease bundle (tree-shaking and stuff) and include only the `version` field of `package.json`.